### PR TITLE
DC-5 evaluations docs

### PR DESCRIPTION
**Pull request by zeryx**
Would love some feedback & maybe some code review if possible.

### CODE REVIEW

File: _pages/algorithm-development/comparing-algorithms.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- Some criteria you want to compare your algorithms with.
+
+Got all that? Great! Lets dive in.
+
+## Step 1: Find your evaluation data
+ ![](https://i.imgur.com/i8Mxzd9.png)
+- If your evaluation requires labelled data comparison, make sure you have the data available and formatted for use in the [data API](https://algorithmia.com/data).
+
+## Step 2: Create an Evaluation (evaluator) algorithm.
+- An evaluator algorithm is special type of algorithm that compares similar algorithms based on some kind of criteria.
```
Comment (StephanieKim) [inline _pages/algorithm-development/comparing-algorithms.md]: "An evaluator algorithm is special" should be "An evaluator algorithm is **a** special"

File: _pages/algorithm-development/comparing-algorithms.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- If you can't find your evaluator, make sure its:
+    - A [published algorithm](https://algorithmia.com/developers/algorithm-development/algorithm-basics/your-first-algo/#publish-your-algorithm)
+    - Visible to your user account
+    - Not created by an Organization (for now)
+
+### Step 4.2: Add your Evaluatable Algorithms
+![](https://i.imgur.com/DaKpEVX.png)
+
+-  If you can't find Evaluatable Algorithm, make sure its:
```
Comment (StephanieKim) [inline _pages/algorithm-development/comparing-algorithms.md]: maybe "can't find **your** Evaluatable Algorithm" IDK kinda reads weird

**Review by stevielkim — CHANGES_REQUESTED** [review]
Thanks so much for doing this and making the changes we talked about earlier! Looks great!

File: _pages/algorithm-development/comparing-algorithms.md (hunk 1 above)
Comment (zeryx) [reply _pages/algorithm-development/comparing-algorithms.md]: thanks! Changed it.

File: _pages/algorithm-development/comparing-algorithms.md (hunk 2 above)
Comment (zeryx) [reply _pages/algorithm-development/comparing-algorithms.md]: updated

**Comment by peckjon** [comment]
Awesome.  Love the screenshots and the way you broke down the steps.  Thinks I think it needs:

- Remove change in tone: first sentence is very informal, remainder is informal. Maybe simply change "So you've been... and you may have..." to "If you have been... you might have..."
- Needs a concrete example up-front (maybe in the intro, before Step 1). You speak in broad generalizations / anything is possible, which is true, and great. But first, ground the audience with a simple end-to-end example, just a few lines like "I take some __ data from ___ for doing ___ / want to compare whether ___ or ___ does it better in [this specific way] / so I write an evaluator algo which takes ___ and compares ___ / then I run it against ___ and ___ and see that [results]"
- Step 1: a bit more detail on how data should be structured? Obvs it can be anything but maybe a mention that it needs to be a format which can be handled by evalueated algos, or does evaluator algo transform it...?
- Step 2: needs more detailed description of how evaluator should be structured, OR if this will be in the reference StringReversal algo, make sure the Docs page for that algo is comprehensive, and link to the docs tab not just the src

Remainder is perfect given our current stage of development of this feature.  Thx!!

**Review by peckjon — APPROVED** [review]
Awesome.  Please remove / comment out the navigation link for now and deploy so we can link into it from upcoming features?

**Pull request merged**